Geometry/HcalTowerAlgo/plugins/HcalTrigTowerGeometryESProducer.cc

```diff
@@ -23,6 +23,10 @@ HcalTrigTowerGeometryESProducer::produce( const CaloGeometryRecord & iRecord )
     bool enableRCTHF=(tmode==HcalTopologyMode::TriggerMode_2009 || tmode==HcalTopologyMode::TriggerMode_2016);
     bool enable1x1HF=(tmode==HcalTopologyMode::TriggerMode_2016 || tmode==HcalTopologyMode::TriggerMode_2017);
     m_hcalTrigTowerGeom->setupHFTowers(enableRCTHF,enable1x1HF);
+    // TODO this piggy-backs on the HF versioning, which corresponds to
+    // 2017 TP --- trigger modes should really be numbered by year.
+    bool enable2017HE = enable1x1HF;
+    m_hcalTrigTowerGeom->setupHETowers(enable2017HE);
```

Comment:
@ianna, not sure I follow completely. Would it be better to move all this into the constructor of the `HcalTrigTowerGeometry`? I could then remove the `setup{HF,HE}Towers` API calls, and the geometry would be static.